Mathlib/GroupTheory/Submonoid/Center.lean

```diff
@@ -97,5 +97,17 @@ end
 
 end Submonoid
 
+variable (M)
+
+/-- For a monoid, the units of the center inject into the center of the units. This is not an
+equivalence in general; one case when it is is for groups with zero, which is covered in
+`centerUnitsEquivUnitsCenter`. -/
+def unitsCenterToCenterUnits [Monoid M] : (Submonoid.center M)ˣ →* Submonoid.center (Mˣ) :=
```

Comment:
```suggestion
@[simps]
def unitsCenterToCenterUnits [Monoid M] : (Submonoid.center M)ˣ →* Submonoid.center (Mˣ) :=
```
or manual lemma(s) if the result is bad.

Reply:
I also added `to_additive`!